packages/diagrams/frontend/sirius-components-diagrams/src/renderer/DiagramRenderer.tsx

```diff
@@ -151,6 +151,9 @@ export const DiagramRenderer = memo(({ diagramRefreshedEventPayload }: DiagramRe
         synchronizeLayoutData(diagramRefreshedEventPayload.id, laidOutDiagram);
       });
     }
+    if (convertedDiagram.style?.background) {
```

Comment:
The style of the diagram cannot be `null` or `undefined` given its TypeScript definition